<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>23</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>QT 2 STM32</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberTemp">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>현재 온도</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSendData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSend">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditRxData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLED1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLED2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxLED3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>92</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LED3</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>현재 습도</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumberWet">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial_servo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial_led">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="notchesVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_Servo">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_led">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Servo PWM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>LED PWM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DHT11 Interval</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_DHT11_interval">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>370</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditSendData</tabstop>
  <tabstop>pushButtonSend</tabstop>
  <tabstop>textEditRxData</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
